.dictionary.txt

```diff
@@ -1,2 +1,4 @@
 !preform
```

Comment:
Commit message: please remember to follow the 50/72 rule and update the pull request description.